asana-helpers-markdown.js

```diff
@@ -111,29 +111,26 @@
     breaks: true,
   });
 
-  document.addEventListener(
-    'paste',
-    (event) => {
+  document.addEventListener('paste', (event) => {
       var editor = getEditor(event.target.parentElement);
 
-      if (editor && !event.isMarkdownFormatEvent && !event.clipboardData.types.contains('text/html')) {
-        event.stopImmediatePropagation();
-        event.stopPropagation();
-        event.preventDefault();
-
-        var pastedText = event.clipboardData.getData('text/plain');
+      if (!editor || event.isMarkdownFormatEvent || !event.clipboardData.types.contains('text/plain')) {
```

Comment:
when pasting images, the event type is `Files`. The previous code was cancelling that, and trying to convert it to markdown.

Now, instead of only ignoring `text/html`, I'm only handling `text/plain`.